web/packages/teleport/src/services/apps/makeApps.ts

```diff
@@ -65,6 +67,9 @@ export default function makeApp(json: any): App {
   if (samlApp) {
     samlAppSsoUrl = `${cfg.baseUrl}/enterprise/saml-idp/login/${name}`;
   }
+  const samlAppPreset = json.samlAppPreset
+    ? json.samlAppPreset
+    : SamlServiceProviderPreset.Unspecified;
```

Comment:
It's probably too late, but I guess it would be simpler to provide a default value on the backend side?
(I'm just not a fan of the `make*` functions, they require us to repeat the same logic for Web UI and Connect on the JS side)

Anyway, we can simplify this assignment:
```ts
  const samlAppPreset = json.samlAppPreset || SamlServiceProviderPreset.Unspecified;
```

Reply:
Its not that late. I may have over thought on new web endpoint hitting on old proxy that wont really return the preset type. I'll remove this default value here, shouldn't be bad for the other place where I would need to handle the saml type.